src-tauri/capabilities/migrated.json

```diff
@@ -0,0 +1,87 @@
+{
+  "identifier": "migrated",
```

Comment:
is there a command called migrated??

Reply:
Capabilities are a new feature that granularly enables and constrains the core exposure to the application front-end. They are essentially a set of [permissions](https://v2.tauri.app/security/permissions/) mapped to application windows. The migrated.js represents the list of capabilities, extracted during the automatic migration step, used in the app.

Reply:
yeah they are new, but not just random.. the identifiers would most of the time refer to commands we expose in invoke handler.. are remember using the windows identifier is as "windows"

Reply:
not because it was added by the migration script that this is the way it should be.. try creating an app with V2 directly.